bokehjs/src/coffee/renderer/guide/categorical_axis.coffee

```diff
@@ -26,6 +26,9 @@ class CategoricalAxis extends Axis.Model
 
     return range_bounds
 
+  display_defaults: () ->
+    return super()
+
```

Comment:
Specifically, see 

https://github.com/bokeh/bokeh/pull/2971/files#diff-6d0b19093e9991d06b237a8b39d3598dR55

which only works when `@type` is an exact match for a block in `default.coffee` So we have to walk up the class hierarchy and this call to super does that. It's not good to need to (remember to) do that, though.